include/ebpf_structs.h

```diff
@@ -10,6 +10,7 @@
 #include <stdbool.h>
 #include <stdint.h>
 #endif
+#include "bpf_structs.h"
```

Comment:
what is the difference between bpf_structs.h vs ebpf_structs.h?  I.e., when should something be added to one vs the other?  Can you add a // comment explaining?  E.g., why is XDP_FLAGS_REPLACE defined in this file and not the other one?  What's the rule?

Reply:
Actually the new file is not needed. Removed it now.